### ISSUE TITLE
the proportion of true and false video is very unbalanced

### ISSUE THREAD

**Issue by IItaly** [polimi-ispl/icpr2020dfdc#53]
Hey, @CrohnEngineer 
the proportion of true and false video is very unbalanced, but it seems that you split it by 1:4. Doesn't it need to be balanced?

**Comment by CrohnEngineer** [comment]
Hey @IItaly ,

which dataset are you talking about? DFDC or FF++?
Bests,

Edoardo

**Comment by CrohnEngineer** [comment]
Anyway, consider that even though both datasets are heavily unbalanced, we have constructed the [FrameFaceIterableDataset](https://github.com/polimi-ispl/icpr2020dfdc/blob/7ae2c9bf7174b8f93040fa425715cb22e9804529/isplutils/data.py#L78) so that the batches are balanced in the number of FAKE and REAL faces, and to stop the training if we have analyzed all the REAL faces.
However, usually the networks overfit long before having analyzed all the REAL faces, and usually the training stops for reaching a plateau on the validation loss or the maximum number of iterations.

**Comment by IItaly** [comment]
Thanks for your reply. I'm talking about FF + + dataset. Because I found that other papers will enhance the real pictures to balance the proportion, so there is such a question.Now I have no problem.

**Comment by CrohnEngineer** [comment]
Glad this solved your doubts :)
Bests,

Edoardo
